OCR.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
```

Comment:
Please add the standard copyright and license header.
Also, this file likely belongs in 'lib' directory vs. top level